From 292df56c579d31479755f5bc8c3dbe3fe935a041 Mon Sep 17 00:00:00 2001
From: Jan Engelhardt <jengelh@inai.de>
Date: Sat, 13 Sep 2014 17:12:43 +0200
Subject: [PATCH 20/21] first_run: barf if autoupdate is activated
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

This patch is for debugging purposes and is meant for discovering
whether — despite our patching efforts — autoupdate is still
enabled somehow, and if so, say so on stderr.

Result: As of October 2014, autoupdate_enabled still is enabled
and a message emitted.
---
 chrome/browser/extensions/extension_system_impl.cc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/chrome/browser/extensions/extension_system_impl.cc b/chrome/browser/extensions/extension_system_impl.cc
index a9d5dbb..d710800 100644
--- a/chrome/browser/extensions/extension_system_impl.cc
+++ b/chrome/browser/extensions/extension_system_impl.cc
@@ -230,6 +230,7 @@ void ExtensionSystemImpl::Shared::Init(bool extensions_enabled) {
   runtime_data_.reset(new RuntimeData(ExtensionRegistry::Get(profile_)));
 
   bool autoupdate_enabled = !profile_->IsGuestSession();
+  fprintf(stderr, "*** autoupdate_enabled = %d\n", autoupdate_enabled);
 #if defined(OS_CHROMEOS)
   if (!extensions_enabled)
     autoupdate_enabled = false;
-- 
2.4.0

